superset-frontend/src/SqlLab/types.ts

```diff
@@ -82,3 +97,31 @@ export interface QueryEditor {
     errors: SupersetError[];
   };
 }
+
+// state refers to superset-frontend/src/components/MessageToasts/getToastsFromPyFlashMessages.js and superset-frontend/src/components/MessageToasts/actions.ts
+export type toastState = {
+  id: string;
+  toastType: ToastType;
+  text: string;
+  duration: number;
+  noDuplicate: boolean;
+};
+
+/** Root state of redux */
```

Comment:
Same as above, I think the code explains well enough without needing a comment.

Reply:
removed. Thank you